src/PostQuery.php

```diff
@@ -127,6 +127,30 @@ public function pagination($prefs = [])
         return $this->pagination;
     }
 
+    /**
+     * Gets the original query used to get a collection of Timber posts.
+     *
+     * @since 2.0
+     * @return WP_Query|null
+     */
+    public function query()
+    {
+        return $this->wp_query;
+    }
+
+    /**
+     * Gets the original query used to get a collection of Timber posts.
+     *
+     * @deprecated 2.0.0, use PostQuery::query() instead.
+     * @return WP_Query|null
+     */
+    public function get_query()
```

Comment:
Maybe start add return types for things we are pretty confident with? `: ?WP_Query`

Reply:
Yes 👍! I added it in ee3537f5d5c54763588f2364fac2ef54042606ef.